client/src/modules/templates/bhYesNoRadios.tmpl.html

```diff
@@ -12,7 +14,7 @@
         ng-value="1"
         ng-model="$ctrl.value"
         ng-change="$ctrl.onChange($ctrl.value)"
-        required>
+        ng-required="$ctrl.required">
```

Comment:
Why do we need to toggle the required if we have a `bhClear`?  It seems like both of them are addressing the same solution.

Reply:
You're right @jniles